interview-questions.md

```diff
@@ -4,57 +4,57 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What does CRUD stand for?
 
-  Your answer:
+  Your answer: Create, Read, Upload, Delete
 
-  Researched answer:
+  Researched answer: "Create, Read, Update, Delete"
 
 
 
 2. What are the 5 HTTP verbs?
 
-  Your answer:
+  Your answer: Put, Get, Delete, Show, Upload???
 
-  Researched answer:
+  Researched answer: Post, Get, Put, Patch, Delete
 
 
 
 3. When creating a basic (stateless) class component in React, what are the necessary elements needed to render "Hello World" on the page?
 
-  Your answer:
+  Your answer: constructor and super
 
-  Researched answer:
+  Researched answer: I'm not sure if I've understood it completely but I believe the basic elements needed are <h1></h1> ??
```

Comment:
this question was a little confusing and your answer is great. but just so you know, we were looking for: import, class name extends, render, return, div/React.Fragment would be a bonus, export included in this answer somehow